test/v1/controllers/requestParsers/AmendBenefitAmountsRequestParserSpec.scala

```diff
@@ -26,10 +26,10 @@ import v1.models.request.AmendBenefitAmounts.{AmendBenefitAmountsRawData, AmendB
 
 class AmendBenefitAmountsRequestParserSpec extends UnitSpec {
 
-  private val nino: String = "AA123456B"
+  private val nino: String    = "AA123456B"
   private val taxYear: String = "2020-21"
-  private val benefitId = "b1e8057e-fbbc-47a8-a8b4-78d9f015c253"
-  implicit val correlationId = "a1e8057e-fbbc-47a8-a8b4-78d9f015c253"
+  private val benefitId       = "b1e8057e-fbbc-47a8-a8b4-78d9f015c253"
+  implicit val correlationId  = "a1e8057e-fbbc-47a8-a8b4-78d9f015c253"
```

Comment:
needs the String type declaration as it's an implicit.
Also ought to make it private, helps speed up compilation when implicits are narrowly scoped